public/spec/lang/2019R2/index.html

```diff
@@ -5837,7 +5837,7 @@ <h3>Annotations</h3>
    <td>type of function</td>
   </tr>
   <tr>
-   <td>resource function</td>
+   <td>resource method</td>
```

Comment:
These shouldn't be changed either.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/6013.